src/DBforsimulation.jl

```diff
@@ -1,9 +1,46 @@
-export get_info_from_DB
+export fix_fknee, get_info_from_DB
 import Random
 
 
 """
-    function get_info_from_DB(db, horns, stokes)
+    function fix_fknee(f_samp_hz, tcmb_k, tatm_k, ttel_k, tnoise_k, β_hz, slope, fknee_bicocca)
+    
+    This function can be used in order to adapt the fknees measured during Bicocca's unit tests to the nominal conditions 
+    of the future data taking in Tenerife.
+    The signal "seen" by the polarimeter in Bicocca was about 21.3 K (22.7 on port A, 29.9 on port B of the Magic Tee as written in
+    Claudio Pincella's Thesis), while in Tenerife will be of about 28.5 (tatm_k + ttel_k + tcmb_k).
+    Therefore, the white noise level will be different in Tenerife, ad so will be the fknee (the 1/f slope, instead is the same).
+
+    This function corrects the fknee according to the following formula:
+
+    fknee_corrected = fknee*(sigma_bicocca/sigma_tenerife)^(1/slope)
+
+    Given:
+    - the sampling frequency, in Hz
+    - the CMB temperature (in K)
+    - the atmospheric temperature, in K
+    - the telescope temperature, in K
+    - the noise temperature of the polarimeter, in K
+    - the bandwidth of the polarimeter, in Hz
+    - the 1/f slope of the polarimeter
+    - the fknee of the polarimeter, as it is from the DB
+
+    It returns the corrected fknee.
+"""
+function fix_fknee(f_samp_hz, tcmb_k, tatm_k, ttel_k, tnoise_k, β_hz, slope, fknee_bicocca)
+
+    τ_s_tenerife = 1/f_samp_hz
+    τ_s_bicocca = 1/25      #f_samp used in Bicocca tests: 25 Hz
+
+    sigma_bicocca = (1/sqrt(2))*(21.3+tnoise_k) / (sqrt(β_hz * τ_s_bicocca))   
```

Comment:
I would not hardcode the values 1/25 s and 21.3 K in the code. Better to create keyword parameters with a default value, so the user can change them if needed:

```julia
function fix_fknee(f_samp_hz, tcmb_k, tatm_k, ttel_k, tnoise_k, β_hz, slope, fknee_bicocca; tref_k = 21.3, fref_samp_hz = 25)
```